### ISSUE TITLE
i got error in the setup 

### ISSUE THREAD

**Issue by CamiloB4A** [ShuhongChen/panic3d-anime-reconstruction#20]
im currently trying to install when i put the command on the terminal in the main directory project i got this error " root@LivingBig:/mnt/f/Avatars/vroid_renderer# docker pull make/docker_pull
Using default tag: latest
Error response from daemon: pull access denied for make/docker_pull, repository does not exist or may require 'docker login': denied: requested access to the resource is denied "

**Comment by ShuhongChen** [comment]
I'm assuming this is an issue for the vroid renderer repo, not the panic3d repo?

The vroid renderer docker tag should be this one here:
https://hub.docker.com/r/shuhongchen/vroid-renderer

Does the docker pull command on the dockerhub page work for you?

**Comment by CamiloB4A** [comment]
let me give you a suggestion for the setup guide, you need to indicate, use this command (docker pull shuhongchen/panic3d-anime-reconstruction; then docker pull shuhongchen/vroid-renderer; ) but im currently stuck with the part of accesing to the container with the run -t Container name because is not clear the commands about build or shell docker

**Comment by ShuhongChen** [comment]
> use this command (docker pull shuhongchen/panic3d-anime-reconstruction; then docker pull shuhongchen/vroid-renderer; )

Supposedly these commands are run by the `make/docker_pull` command, the specific location is [here](https://github.com/ShuhongChen/panic3d-anime-reconstruction/blob/d2cb8a0abbb523a91809d1522f6c44131ff14cec/make/_make.sh#LL41C11-L41C11)

Maybe you can `echo` there to make sure the variables are set properly?

> but im currently stuck with the part of accesing to the container with the run -t Container name because is not clear the commands about build or shell docker

Sorry I don't quite understand what you mean by this could you rephrase
